include/exadg/incompressible_navier_stokes/spatial_discretization/operator_coupled.h

```diff
@@ -73,6 +73,16 @@ class NonlinearOperatorCoupled
     pde_operator->evaluate_nonlinear_residual(dst, src, rhs_vector, time, scaling_factor_mass);
   }
 
+  /*
```

Comment:
this is the function added for the 'operator_coupled' and 'operator_pressure_correction' and for the structure module (empty/asserted in the latter case).